releasenotes/notes/support-sprase-pauli-op-00d29000d48c93ca.yaml

```diff
@@ -0,0 +1,13 @@
+---
+features:
+  - |
+    Updated :func:`~.to_ising` to support ``qiskit.quantum_info.SparsePauliOp``.
+    Added ``opflow`` argument to :func:`~.to_ising` to control the output is
+    opflow operator (if ``True``) or ``SparsePauliOp`` (if ``False``).
+    The default value of ``opflow`` argument is currently ``True``, but it will
+    first be changed to ``False`` and then deprecated in future releases.
```

Comment:
```suggestion
    Updated :func:`~.to_ising` to support optionally returning ``qiskit.quantum_info.SparsePauliOp``.
    This is achieved by a new ``opflow`` argument to :func:`~.to_ising` to control if the output is
    the opflow operator, as has been done in the past,  (if ``True``) or ``SparsePauliOp`` (if ``False``).
    The default value of ``opflow`` argument is currently ``True``, but it will
    first be changed to ``False``. The parameter will be deprecated and removed in a future release.
```

Reply:
Thanks. I include it. cc41551